webclient/projekt/doc_utils.py

```diff
@@ -239,336 +291,184 @@ class OznameniPDFCreator(DocumentCreator):
-            datum_zmeny = self.projekt.historie.historie_set.first().datum_zmeny
-        else:
-            datum_zmeny = datetime.datetime.today()
+        self._create_data_document_part()
 
         self.texts[
             "doc_par_1"
         ] = f"""
-        Potvrzujeme, že <strong>{self.oznamovatel.oznamovatel}</strong> ({self.oznamovatel.adresa}; tel: {self.oznamovatel.telefon}; \n\
-        email: {self.oznamovatel.email}), jehož zastupuje {self.oznamovatel.odpovedna_osoba}, \n\
-        ohlásil záměr <strong>{self.projekt.podnet}</strong> (označení stavby: {self.projekt.oznaceni_stavby}; \n\
-        plánované zahájení: {self.projekt.planovane_zahajeni.lower.strftime('%d. %m. %Y').replace(' 0', ' ') if self.projekt.planovane_zahajeni else ''} - \n\
-        {self.projekt.planovane_zahajeni.upper.strftime('%d. %m. %Y').replace(' 0', ' ') if self.projekt.planovane_zahajeni else ''} na \n\
-        k. ú. <strong>{str(self.projekt.hlavni_katastr).replace("(", "(okr. ")}</strong>, parc. č. \n\
-        {self.projekt.parcelni_cislo} ({self.projekt.lokalizace}) {DOC_KOMU[self.dok_index]}. \n\
-        Oznámení provedl <strong>{datum_zmeny.strftime('%d. %m. %Y').replace(' 0', ' ')}</strong> pod evidenčním číslem \n\
-        <strong>{self.projekt.ident_cely}</strong>. <strong>Tímto byla naplněna povinnost oznámit \n\
-        zamýšlenou stavební nebo jinou činnost Archeologickému ústavu podle ustanovení § 22, odst. 2, zákona \n\
-        č. 20/1987 Sb. Po schválení a registraci oznámení konkrétní organizací oprávněnou provádět archeologické \n\
-        výzkumy Vás budeme informovat.</strong>
+        Archeologický ústav AV ČR, {DOK_MESTO[self.dok_index]}, v. v. i., obdržel dne 
+        {self.projekt.datum_oznameni.strftime('%d. %m. %Y').replace(' 0', ' ') if self.projekt.datum_oznameni else ''} 
+        Vaše oznámení o zahájení stavebního (či jiného) záměru na území s archeologickými nálezy 
+        {self.projekt.podnet} v k. ú. {self.projekt.hlavni_katastr} ({self.projekt.hlavni_katastr.okres}), který 
+        eviduje pod evidenčním číslem {self.projekt.ident_cely}. Archeologický ústav AV ČR, 
+        {DOK_MESTO[self.dok_index]}, v. v. i. sděluje, že tímto krokem byla splněna zákonná oznamovací povinnost 
+        dle ustanovení § 22 odst. 2 zákona č. 20/1987 Sb., o státní památkové péči, ve znění pozdějších předpisů.
         """
 
         self.texts[
             "doc_par_2"
-        ] = """TENTO DOKUMENT NESLOUŽÍ JAKO POTVRZENÍ O PROVEDENÍ ARCHEOLOGICKÉHO \n\
-            VÝZKUMU PRO STAVEBNÍ ČI JINÉ ŘÍZENÍ."""
-
-        self.texts[
-            "doc_par_3"
-        ] = """
-        Oznamovatel je v návaznosti na oznámení povinen umožnit Archeologickému ústavu nebo oprávněné organizaci \n\
-        provést na dotčeném území záchranný archeologický výzkum. O tomto mu bude potvrzení vystaveno samostatně, \n\
-        a to organizací, která výzkum realizovala.
-        """
-
-        self._create_signature()
-
```

Comment:
Odtud dále to mělo být zachováno jako příloha. Prosím vrátit zpět celou přílohu k osobním údajům.

Reply:
A co myslíš přílohou? Jako že to má být nějaký jiný dokument?

Reply:
Jde o to, co je zde od str. 2 dále: [oznameni_C-202213997_20250302123523.pdf](https://github.com/user-attachments/files/19042803/oznameni_C-202213997_20250302123523.pdf)

Tj. je to součást dokumentu, ale až za podpisem a začíná to na nové straně.

Reply:
Díky za vysvětlení, opraveno.

Reply:
Já mám pocit, že se stále nechápeme. Má to být tak, že bude v dokumentu celý obsah https://docs.google.com/document/d/1oGJDHAaLp3RRQX_iAP8K3brqFHriK4n8hxkrpCniEn8/edit?tab=t.0 a potom zůstane z původní verze část "PŘÍLOHA – INFORMACE O ZPRACOVÁNÍ OSOBNÍCH ÚDAJŮ" (tj. vše od tohoto komentáře dále z původní verze).

Reply:
Tak snad už je to ok.